etc/Syncer.json

```diff
@@ -353,7 +487,10 @@
           },
           "aliasColors": {},
           "bars": false,
-          "datasource": "${DS_BIGDATA-CLUSTER}",
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "${DS_TIDB-CLUSTER}",
+          "description": "the number of different binlog files between the upstream and the downstream in the process of synchronization; the normal value is 0, which indicates real-time synchronization; a larger value indicates a larger number of binlog files discrepancy",
```

Comment:
```suggestion
          "description": "the number of different binlog files between the upstream and the downstream in the process of replication; the normal value is 0, which indicates real-time replication; a larger value indicates a larger number of binlog files discrepancy",
```

Reply:
don't need to review `Syncer.json`, I would update from `syncer.md` later